### DISCUSSION TITLE
2024/09/12/AI.%E5%AD%A6%E4%B9%A0%E6%9E%B6%E6%9E%84%E6%A2%B3%E7%90%86/

### DISCUSSION THREAD

**Discussion by giscus** [Jared-ZDC/markel#39] [General]
# 2024/09/12/AI.%E5%AD%A6%E4%B9%A0%E6%9E%B6%E6%9E%84%E6%A2%B3%E7%90%86/

摘要最近要把AI 整体自底向上的梳理一遍，同时把自己的知识面好好扩充一下，因此需要整体的看看，主要有哪些东西，并且这些东西，应该从哪些地方去补充知识； 坦白说，这部分跟我自己以往的工作经历差别有点大，从通用服务器芯片的全栈，要转移到AI专用芯片的全栈，这个里面涵盖的知识面可能相对通用服务器要求的更高，所以这里可能近一年需要花比较多的功夫；   雄关漫道真如铁，而今迈步从头越

https://markel.top/2024/09/12/AI.%E5%AD%A6%E4%B9%A0%E6%9E%B6%E6%9E%84%E6%A2%B3%E7%90%86/

**Comment by Jared-ZDC** [comment]
梳理业界当前的AI\GPU 芯片的架构分析总结 
主要的芯片以及架构有：
* 特斯拉FSD芯片
* 昇腾芯片
* Navidia 芯片roadmap 
<img src="https://inews.gtimg.com/news_bt/OyEw4lG72eufHBb6UROuyfouBlejOReSxK5owcEVuo5HcAA/641"/>

* google TPU
